keras_cv/layers/preprocessing/solarization.py

```diff
@@ -49,16 +58,31 @@ class Solarization(tf.keras.layers.Layer):
             or [height, width, channels].
     """
 
-    def __init__(self, threshold=None):
+    def __init__(
+        self,
+        addition=0.0,
+        threshold=None,
```

Comment:
if the value should be int or float, should we have default value to be 0.0?

Reply:
Fixed